app/js/lib/foxbox/settings.js

```diff
@@ -187,6 +188,14 @@ export default class Settings extends EventDispatcher {
     return ONLINE_CHECKING_LONG_INTERVAL;
   }
 
+  /**
+   * Minimal interval between consequent value watcher requests.
+   * @return {number}
+   */
+  get watchInterval() {
```

Comment:
If this watcher idea sticks, we can later implement new setting to enable/disable watching, but good thing that in most common conditions like ```box is not available``` or ```user is not authenticated``` our current API won't try to make any call anyway.